File: PolicyManagement.Api/PolicyManagement.Models/Motor/PolicyTermFormDataModel.cs
using PolicyManagement.Dtos.Common;
using System;

namespace PolicyManagement.Models.Motor
{
    public class PolicyTermFormDataModel
    {
        public short PolicyType { get; set; }
        public short VehicleClass { get; set; }
        public string PackageType { get; set; }
        public short PackageTypeId { get; set; }
        public short PolicyTerm { get; set; }
        public string AcknowledgementSlipNumber { get; set; }
        public string AcknowledgementSlipIssueDateString { get; set; }
        public DateTime? AcknowledgementSlipIssueDate { get; set; }
        public DateDto AcknowledgementSlipIssueDateDto { get; set; }
    }
}

File: PolicyManagementAngular/src/app/app-entites/models/motor/policy-term-form-data-model.ts
import { IDateDto } from "src/app/app-entites/dtos/common/date-dto";

export interface IPolicyTermFormDataModel {
    PolicyType: number;
    VehicleClass: number;
    PackageType: string;
    PackageTypeId: number;
    PolicyTerm: number;
    AcknowledgementSlipNumber: number;
    AcknowledgementSlipIssueDateString: string;
    AcknowledgementSlipIssueDateDto: IDateDto | null;
}